byterun/caml/misc.h

```diff
@@ -140,6 +234,11 @@ extern void caml_ext_table_remove(struct ext_table * tbl, void * data);
 extern void caml_ext_table_free(struct ext_table * tbl, int free_entries);
 extern void caml_ext_table_clear(struct ext_table * tbl, int free_entries);
 
+CAMLextern int caml_read_directory(char * dirname, struct ext_table * contents);
+
+
+#ifdef CAML_INTERNALS
```

Comment:
Is it on purpose that definitions just above (caml_ext_table) are no longer protected by CAML_INTERNALS?